core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/js/JsModifier.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Automated Exploratory Tests
+ * <p>
+ * Author: pnad@github
+ * used code - Copyright (C) 2017 Cognifide Limited
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.aet.job.common.modifiers.js;
+
+import com.cognifide.aet.communication.api.metadata.CollectorStepResult;
+import com.cognifide.aet.job.api.ParametersValidator;
+import com.cognifide.aet.job.api.collector.CollectorJob;
+import com.cognifide.aet.job.api.collector.CollectorProperties;
+import com.cognifide.aet.job.api.exceptions.ParametersException;
+import com.cognifide.aet.job.api.exceptions.ProcessingException;
+
+import org.apache.commons.lang3.BooleanUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+
+public class JsModifier implements CollectorJob {
+
+    public static final String NAME = "js";
```

Comment:
I don't agree. We already have [js-errors](https://github.com/Cognifide/aet/wiki/JSErrorsCollector) collector. It may be misleading having `js` tag.
I suggest renaming `cmd` param  to `script` also.

About long script - I thought about this some time ago and there is just an idea, how about providing script as a web resource and `script` value should be just a link for it?
E.g.
```xml
<execute-js script="http://my-site-to-test/aet/some-script.js" />
```

This solution has its disadvantages, however it will not obscure the suite XML definition.